Updated exclude.yml for MT protocol

valosekj opened the pull request:
Added fslAchieva04 and beijingPrisma01 under `mtsat` due to poor data quality (https://github.com/spine-generic/data-multi-subject/issues/36#issuecomment-686536166).
Added information about bad TR for fslAchieva03-06 under `t1` (https://github.com/spine-generic/data-multi-subject/issues/36#issuecomment-687112400)
Added sub-beijingPrisma01 under `t1` due to poor data quality (https://github.com/spine-generic/data-multi-subject/issues/36#issuecomment-686536166)

Should not we also exclude fslAchieva03-06 subjects with bad t1 TR also for `mtsat` when t1 is used for mtsat calculation?

https://github.com/spine-generic/spine-generic/blob/f07cde93193160f62a4e04f47c31a83d14b0f0a7/process_data.sh#L225-L226

Related to: https://github.com/spine-generic/data-multi-subject/issues/36

valosekj committed:
mtsat and t1 update

Added fslAchieva04 and beijingPrisma01 under mtsat due to poor data quality (https://github.com/spine-generic/data-multi-subject/issues/36#issuecomment-686536166). Added information about bad TR for fslAchieva03-06 under t1. Added sub-beijingPrisma01 under t1 due to poor data quality.

jcohenadad reviewed (commented):
We need to do further investigations before removing sites for mtsat/t1: https://github.com/spine-generic/data-multi-subject/issues/36#issuecomment-688535998

jcohenadad committed:
Excluded fslAchieva

jcohenadad committed:
Moved DTI metrics at the bottom; added MD and RD

valosekj commented on exclude.yml:
```diff
@@ -40,3 +35,18 @@ t1:
+- sub-mountSinai03  # T2w was re-acquired (hence wrong T2w -> DWI registration)
+- sub-oxfordFmrib04  # T1w scan is not aligned with other contrasts (subject repositioning)
+- sub-oxfordFmrib01  # registration issue (segmentation OK)
+fig_dti_rd:
```
Should not be?:
```suggestion
dti_rd:
```

jcohenadad committed:
Fixed typo

Co-authored-by: Jan Valošek <39456460+valosekj@users.noreply.github.com>

jcohenadad committed:
Harmonized exclusion across mtr, mtsat, t1

jcohenadad committed:
Merge branch 'master' into patch-1

jcohenadad commented:
@valosekj there are still some changes i'd like to bring  (don't merge now)

jcohenadad committed:
Removed sub-fslAchieva03-06 due to poor data quality

jcohenadad committed:
Excluded sub-pavia05

Added image: https://github.com/spine-generic/data-multi-subject/issues/36

jcohenadad committed:
Excluded sub-nottwil03

Image added: https://github.com/spine-generic/data-multi-subject/issues/36

jcohenadad committed:
Re-included pavia05 and nottwil03

It sheds interesting light about the reliability of MTsat wrt. MTR, which should be discussed in the paper
